scripts/plots/analyses/exevs_analyses_grid2obs_plots.sh

```diff
@@ -165,6 +165,25 @@ do
 	then
 	echo "WARNING: NO PLOT FOR",$varb,$region,$anl
         fi
+
+	if [ ! -e $COMOUTplots/$varb/evs.${anl}.bcrmse_me.${smvar}_${smlev}.last31days.timeseries.buk_${smregion}.png ]
+	then
+	$PARMevs/metplus_config/${STEP}/${COMPONENT}/${VERIF_CASE}/py_plotting.config_timeseries
+	export err=$?; err_chk
+        else
+	echo "RESTART - plot exists; copying over to plot directory"
+	cp $COMOUTplots/$varb/evs.${anl}.bcrmse_me.${smvar}_${smlev}.last31days.timeseries.buk_${smregion}.png $PLOTDIR
+	fi 
+
+	if [ -e ${PLOTDIR}/sfc_upper/*/evs*png ]
+	then
+        mv ${PLOTDIR}/sfc_upper/*/evs*png ${PLOTDIR}/evs.${anl}.bcrmse_me.${smvar}_${smlev}.last31days.timeseries.buk_${smregion}.png
```

Comment:
The wildcards look like it is moving a bunch of files but it looks like it is renaming one file? If it is just 1 file being renamed can we use the full file name and not wildcards?

Reply:
I'd have to work on this because it is a long name that I'd have to figure out how to construct.  This has been used for EVSv1, so I assume it's been well vetted.  If this is a necessary change, I'd like to do this in a new PR later because of the scope of the changes and the number of lines that would need to be changed.